content/docs/pipelines/sdk/install-sdk.md

```diff
@@ -73,7 +73,17 @@ up Python using [Miniconda](https://conda.io/miniconda.html):
 Run the following command to install the Kubeflow Pipelines SDK:
 
 ```bash
-pip install https://storage.googleapis.com/ml-pipeline/release/latest/kfp.tar.gz --upgrade
+pip3 install kfp --upgrade
```

Comment:
That is a really good point... 
>pip alias needs to correspond to the python alias
this is correct, however if we use pip when users do not have a conda / virtual env it will fail as pip will point to python2 by default. 

Good news is that there is a work around that will bypass both issues 

python3 -m pip install --upgrade kfp

This will always install where your python points to. I'm upgrading the instructions accordingly. Also could you verify and let me know if this works for your env as well.